source/propOr.js

```diff
@@ -22,11 +21,19 @@ import pathOr from './pathOr';
  *        name: 'ALICE',
  *        age: 101
  *      };
+ *      const bob = {
+ *        name: 'BOB',
+ *        age: 61,
+ *        favoriteLibrary: 'Ramda@0.26.0'
+ *      };
  *      const favorite = R.prop('favoriteLibrary');
  *      const favoriteWithDefault = R.propOr('Ramda', 'favoriteLibrary');
  *
  *      favorite(alice);  //=> undefined
  *      favoriteWithDefault(alice);  //=> 'Ramda'
+ *      favoriteWithDefault(Object.create(alice));  //=> 'Ramda'
```

Comment:
What is this line trying to prove?  It seems to say that if Alice doesn't have the property neither will a clone of Alice; that shouldn't be a surprise.

While I can at least conceive of a reason to do this for Bob, where the value will be found on the prototype, I can see no reason to do it for Alice.

Reply:
I think this one line should simply be removed.

Reply:
You're right, this is nonsense in this form.

What I'd like to document is that it _does_ check the prototype for the property. This is where the behaviour has changed compared to version 0.25, which only considered own properties. Would it be more useful if I write the following (as the last example):

```js
favoriteWithDefault(Object.create(bob));  //=> 'Ramda@0.26.0'
```

Reply:
While I think that would be better, I'm not convinced that it's necessary.  Our examples are not meant to be exhaustive, nor to show version changes.  However, I wouldn't object if you feel strongly about it.